cmd/register/files.go

```diff
@@ -91,9 +106,23 @@ func Register(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext)
 	logger.Infof(ctx, "Parsing files... Total(%v)", len(dataRefs))
 	fastFail := !rconfig.DefaultFilesConfig.ContinueOnError
 	var registerResults []Result
+
 	for i := 0; i < len(dataRefs) && !(fastFail && _err != nil); i++ {
-		registerResults, _err = registerFile(ctx, dataRefs[i], registerResults, cmdCtx)
+		if strings.Contains(dataRefs[i], ".tar.gz") && !rconfig.DefaultFilesConfig.FastRegister {
```

Comment:
We support archive files for regular register aswell. 
This condition will remove that functionality